doc/contributing/releases.md

```diff
@@ -90,10 +90,11 @@ responsible for that release. In order to be able to verify downloaded binaries,
 the public should be able to check that the `SHASUMS256.txt` file has been
 signed by someone who has been authorized to create a release.
 
-The GPG keys should be fetchable from a known third-party keyserver. The SKS
-Keyservers at <https://sks-keyservers.net> are recommended. Use the
-[submission](https://pgp.mit.edu/) form to submit a new GPG key. You'll need to
-do an ASCII-armored export of your key first:
+The GPG keys should be fetchable from a known third-party keyserver.
+The OpenGPG keyserver at <https://keys.openpgp.org/> is recommended.
+Use the [submission](https://keys.openpgp.org/upload) form to submit
+a new GPG key, and make sure to verify the associated email.
```

Comment:
```suggestion
a new PGP key, and make sure to verify the associated email.
```

Reply:
I think GPG key is correct

Reply:
It feels quite weird to call it a GPG key, when GPG is just the client, and the key is not GPG-specific (it would work with any PGP client).
I tried to search through the GnuPG docs to see if they'd call them one way or another, but no, they just call them "keys". Trying to display the key with `gpg` does mention PGP, but not "PGP key" (nor "GPG key"):
```console
$ gpg --armor --export CC68F5A3106FF448322E48ED27F5E38D5B0A215F | head -1
-----BEGIN PGP PUBLIC KEY BLOCK-----
```
Maybe we should simply call it "public key"?
```suggestion
a new public key, and make sure to verify the associated email.
```